### PR TITLE
docs(guides): add remix guide page

### LINKED ISSUE #63: (docs): Getting started with Remix
### Subject

Getting Started

### Description

Now that [Remix](https://remix.run/docs/en/v1) is open sourced I think it would make sense to create a section to get started started with Remix and Chakra UI.

### PR DESCRIPTION
**Pull request by noobinthisgame**
Closes #63 

## 📝 Description

Adds a guide to get started with Remix.

## 💣 Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by noobinthisgame** [comment]
@lazarnikolov94 Please lmk if you have something to add to this Remix guide.

**Comment by lazarnikolov94** [comment]
I haven't gotten into Remix that much just yet, so I don't really know if there are any additional stuff to mention. I would suggest adding a little blockquote at the top saying that this document might be a "work-in-progress" and call out people who know Remix to contribute to the documentation.

**Review by lazarnikolov94 — CHANGES_REQUESTED** [review]
I haven't gotten into Remix that much just yet, so I don't really know if there are any additional stuff to mention. I would suggest adding a little blockquote at the top saying that this document might be a "work-in-progress" and call out people who know Remix to contribute to the documentation.

**Comment by noobinthisgame** [comment]
Alright :) Good idea will do so.

**Comment by noobinthisgame** [comment]
Good catch thanks! Will change that :)

**Comment by lazarnikolov94** [comment]
@noobinthisgame let me know when it's ready to be reviewed again

**Comment by noobinthisgame** [comment]
@lazarnikolov94 it should be ready with your suggestions implemented

**Comment by lazarnikolov94** [comment]
Awesome! I'll do a complete review tomorrow. Looks great btw!

**Comment by lazarnikolov94** [comment]
> @lazarnikolov94 it should be ready with your suggestions implemented

Awesome! I'll do a complete review tomorrow. Looks great btw!

**Comment by noobinthisgame** [comment]
Cool thanks!!!

**Comment by lazarnikolov94** [comment]
I see @primos63 wrote a good guide on Remix here: https://github.com/chakra-ui/chakra-ui/issues/5166 Maybe you could apply these changes and update this PR.

**Comment by noobinthisgame** [comment]
@lazarnikolov94 I added more details to the provider setup as suggested by @primos63.

File: pages/guides/getting-started/remix-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+```bash
+npm i @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^4
+```
+
+```bash
+yarn add @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^4
+```
+
+> 🚨 As of now yarn is not officially supported in Remix projects, use with care!
```
Comment (lazarnikolov94) [inline pages/guides/getting-started/remix-guide.mdx:24]: > 🚨 Right now yarn is not officially supported in Remix projects, so use it with care!

File: pages/guides/getting-started/remix-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,151 @@
+---
+title: Remix
+description: A guide for installing Chakra UI with Remix projects
+tags: ['remix']
+author: noobinthisgame
+---
+
+> This guide is still work in progress! Feel free to contribute if you have
+> built something with Remix and Chakra UI 😃
```
Comment (lazarnikolov94) [inline pages/guides/getting-started/remix-guide.mdx:9]: > This guide is still work in progress! Feel free to contribute if you have
> something built with Remix and Chakra UI 😃

**Review by lazarnikolov94 — CHANGES_REQUESTED** [review]
Just minor word play. Looks good btw 👌

**Comment by noobinthisgame** [comment]
@lazarnikolov94 texts are changed :)

**Pull request merged**